app/services/idv/financials_step.rb

```diff
@@ -29,6 +29,10 @@ def vendor_errors
       vendor_validator.errors if form_valid?
     end
 
+    def vendor_reasons
+      vendor_validator.reasons if form_valid?
```

Comment:
What do you think about consolidating the bad reasons into the errors hash, and display the good reasons only in the success scenario, and call it something like `vendor_success_details`?